errata_tool/cli/variants.py

```diff
@@ -0,0 +1,24 @@
+from errata_tool.variants import Variants
```

Comment:
Let's drop this new `variants` cli from this particular pull request, since it's not directly related to extracting the data for use with Ansible, and I want to think through the UX here a bit more (`get` vs `list`, and maybe making this a `product-version` thing instead)